.github/workflows/main.yml

```diff
@@ -37,7 +45,7 @@ jobs:
 
     - name: Install Build Tools (apt)
       run: >
-        sudo apt update
+        sudo apt update || true
```

Comment:
What's up with that?

Reply:
Sometimes, `apt update` will catch a mirror in the middle of a mirror sync and spit out a useless error. It's mostly harmless, since the apt cache is still valid and you can still install packages (especially since GitHub keeps their images up to date), but it does throw an `exit 1` and error out the CI job. It's a rare thing to happen but it has happened once or twice while testing CI, so I figured it was worth a quick workaround to avoid it.

Reply:
Ah, so it's debian being garbage as usual, ok.

> since the apt cache is still valid and you can still install packages (especially since GitHub keeps their images up to date)

Does this mean the images already come with a reasonably up to date apt cache? If so, do we even need `apt update` at all?

Reply:
It's still a good practice to do it, since sometimes a random package will be deleted from their download stores, i.e: in the case of a severe CVE where they don't want anyone downloading it if possible. It's doubly-rare that both `apt update` *and* a package would be deleted, so I think it's fine for now.

Reply:
A brief comment explaining the purpose of it would be nice, then.

Reply:
Done.